hosts/juno/jiram/img.py

```diff
@@ -4,22 +4,20 @@
 
 import numpy as np
 import julian
-import pdstable
 import cspyce
 from polymath import *
 import os.path
-import pdsparser
 import oops
 
-from . import JIRAM
+from hosts.juno.jiram import JIRAM
```

Comment:
The problem is with '.' you can't run this module's main() using:
% python img.py

Instead you have to use:
% python -m hosts.juno.jiram.img

This is a bit more of a pain to deal with.  Also (though perhaps not relevant in this case), it wouldn't be interpreted correctly in a shebang because the -m would get bundled with the command instead of being an argument to python.  Also, pep-8 discourages relative imports, though it does allow for the form you mention above: '.juno.jiram'